care/facility/migrations/0106_auto_20200510_1257.py

```diff
@@ -0,0 +1,274 @@
+# Generated by Django 2.2.11 on 2020-05-10 07:27
+
+from django.db import migrations, models
+import uuid
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "import uuid" should be placed before "from django.db import migrations, models"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5498155)